src/platforms/common/configuration/options.mdx

```diff
@@ -352,7 +352,13 @@ Maximum number of characters a single value can have before it will be truncated
 
 <ConfigKey name="normalize-depth" supported={["javascript", "node"]}>
 
-Sentry SDKs normalize any contextual data to a given depth. Any keys containing data with a structure deeper than this will be trimmed and marked using its type instead (`[Object]` or `[Array]`), without walking the tree any further. By default, walking is performed `3` levels deep.
+Sentry SDKs normalize any contextual data to a given depth. Any data beyond this depth will be trimmed and marked using its type instead (`[Object]` or `[Array]`), without walking the tree any further. By default, walking is performed `3` levels deep.
+
+</ConfigKey>
+
+<ConfigKey name="normalize-max-depth" supported={["javascript", "node"]}>
+
+This is the maximum number of properties or entries which will be included in any given object or array when the SDK is normalizing contextual data. Any data beyond this depth will be dropped. (defaults to 1000)
```

Comment:
```suggestion
This is the maximum number of properties or entries that will be included in any given object or array when the SDK is normalizing contextual data. Any data beyond this depth will be dropped. (defaults to 1000)
```

Reply:
Ugh. This is literally the one grammar thing where the wrong way is what sounds more natural to my ears. I'll try to pay more attention.

Reply:
You may have somehow picked up on a British English way of doing it. My Aussie tech writing friend informed me the other day that we do this differently. It's a bit more interchangeable in British English apparently.